protocol/binary_test.go

```diff
@@ -152,6 +184,22 @@ func TestI8EOFFailure(t *testing.T) {
 	checkEOFError(t, wire.TI8, tests)
 }
 
+func TestStreamI8(t *testing.T) {
+	tests := []streamEncodeDecodeTest{
+		{"0", testvalue(vi8(0)), []byte{0x00}},
+		{"1", testvalue(vi8(1)), []byte{0x01}},
+		{"-1", testvalue(vi8(-1)), []byte{0xff}},
+		{"127", testvalue(vi8(127)), []byte{0x7f}},
+		{"-128", testvalue(vi8(-128)), []byte{0x80}},
+	}
```

Comment:
Instead of duplicating the test data, is it possible to add the `description` to `encodeDecodeStruct` and call `testvalue` upon passing `test.value` into `checkStreamEncode`?

Alternatively, was there no way to use `wire.Value.Get()`?